include/libxnvme.h

```diff
@@ -30,6 +30,7 @@ extern "C" {
 #include <sys/types.h>
 #include <sys/uio.h>
 #include <time.h>
+#include <sys/eventfd.h>
```

Comment:
You can move this include into ``xnvme_be_vfio_async.c`` where ``eventfd()`` is called.